modules/ROOT/pages/tools/neo4j-admin/migrate-configuration.adoc

```diff
@@ -57,16 +58,16 @@ The following example shows how to migrate a legacy configuration file to the cu
 
 [source, shell, subs="attributes+"]
 ----
-bin/neo4j-admin server migrate-configuration --from-path=/path/to/legacy/neo4j-enterprise-4.4.10/conf/ --to-path=/path/to/new/neo4j-enterprise-{neo4j-version-exact}/conf/
+bin/neo4j-admin server migrate-configuration --from-path=/path/to/legacy/neo4j-enterprise-4.4.39/conf/ --to-path=/path/to/new/neo4j-enterprise-{neo4j-version-exact}/conf/
 ----
 
 .Example output
 [result]
 ----
-Keeping original user-logs.xml file at: /neo4j-enterprise-5.1.0/conf/user-logs.xml.old
-User logging configuration xml file generated: /neo4j-enterprise-5.1.0/conf/user-logs.xml
-Keeping original server-logs.xml file at: /neo4j-enterprise-5.1.0/conf/server-logs.xml.old
-Server logging configuration xml file generated: /neo4j-enterprise-5.1.0/conf/server-logs.xml
+Keeping original user-logs.xml file at: /neo4j-enterprise-2025.1.0/conf/user-logs.xml.old
```

Comment:
2025.1.0 or 2025.01.0?
Same question for lines 68, 69, 70, and 93

Reply:
Ah, yes. Good spot.